lib.sh

```diff
@@ -76,9 +76,10 @@ use_global_systemd_dns
 # Whiptails
 TITLE="Nextcloud VM - $(date +%Y)"
 [ -n "$SCRIPT_NAME" ] && TITLE+=" - $SCRIPT_NAME"
-CHECKLIST_GUIDE="Navigate with the [ARROW] keys and (de)select with the [SPACE] key. Confirm by pressing [ENTER]"
-MENU_GUIDE="Navigate with the [ARROW] keys and confirm by pressing [ENTER]"
-RUN_LATER_GUIDE="You can view this script later by running 'sudo bash $SCRIPTS/menu.sh'"
+CHECKLIST_GUIDE="Navigate with the [ARROW] keys and (de)select with the [SPACE] key. \
+Confirm by pressing [ENTER]. Cancel by pressing [Esc]."
```

Comment:
Have you tested this, does ESC really work?

Please use CAPITAL letters.

Thanks!

Reply:
OK, ESC works.